src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(callback) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(callback = compareFunction) {
```
let's write default compare function and include it as a default value for your callback here